backend/app/models/model_evaluation.py

```
@@ -36,60 +36,108 @@ def dedupe_ids(cls, v: list[int]) -> list[int]:
+        sa_column_kwargs={"comment": "Timestamp when the evaluation was last updated"},
+    )
+    deleted_at: datetime | None = Field(
+        default=None,
+        nullable=True,
+        sa_column_kwargs={"comment": "Timestamp when the evaluation was deleted"},
+    )
 
-    project: "Project" = Relationship()
+    # Relationships
+    project: Project = Relationship()
     fine_tuning: "Fine_Tuning" = Relationship(back_populates="model_evaluation")
```

Ruff (0.14.8)

Undefined name `Fine_Tuning`

(F821)